webrtc.html

```diff
@@ -1942,6 +1942,28 @@ <h4>Set the RTCSessionDescription</h4>
                       </li>
                     </ol>
                   </li>
+                  <li>
+                    <p>If <var>description</var> is of type <code>"answer"</code>, then run
+                    the following steps:</p>
+                    <ol>
+                      <li>
+                        <p>For each <var>transceiver</var> in the <var>connection</var>'s
+                        <a href="#transceivers-set">set of transceivers</a> run the
+                        following steps:</p>
+                        <ol>
+                          <li>
+                            <p>If <var>transceiver</var> is <a data-link-for="RTCRtpTransceiver">
+                            stopped</a>, it is <a>associated</a> with an m= section, and the
```

Comment:
Remove comma

Reply:
If the transceiver is stopped, associated with an m= section and the associated m= section is rejected...

Reply:
Done.